packages/jsii-dotnet-generator/src/Amazon.JSII.Generator.UnitTests/AssemblyGeneratorTests.cs

```diff
@@ -352,6 +363,115 @@ public void CreatesProjectFileWithDependencies()
             file.Received().WriteAllText(projectFilePath, Arg.Do<string>(actual => PlatformIndependentEqual(expected, actual)));
         }
 
+        [Fact(DisplayName = Prefix + nameof(CreatesAnchorFile))]
+        public void CreatesAnchorFile()
```

Comment:
_ CreateAnchorFile

Reply:
Verb is actually correct for a test.

Reply:
This method doesn't create an anchor file, it's a test method. So the title means `"Verify that the assembly generator creates the anchor file."`
`won'tfix`